libs/SalesforceSDKCore/SalesforceSDKCore/Classes/UserAccount/SFUserAccountManager.m

```diff
@@ -653,7 +652,7 @@ - (void)logoutUser:(SFUserAccount *)user reason:(SFLogoutReason)reason {
     
     // Before starting actual logout (which will tear down SFRestAPI), first unregister from push notifications if needed
     __weak typeof(self) weakSelf = self;
-    [[SFPushNotificationManager sharedInstance] unregisterSalesforceNotificationsWithCompletionBlock:user completionBlock:^void() {
+    (void)[[SFPushNotificationManager sharedInstance] unregisterSalesforceNotificationsWithCompletionBlock:user completionBlock:^void() {
```

Comment:
can the void go away here too now?

Reply:
ah I missed one.  Sorry. I've been doing to much context shifting :D .  I'll fix that.